docs/plugins/3dveins.rst

```diff
@@ -1,6 +1,9 @@
 3dveins
 =======
-**Tags:** `tag/fort`, `tag/mod`, `tag/map`
+
+.. dfhack-tool::
+    :tags: fort, mod, map
```

Comment:
Worth noting that with how I've written `directive_arg_str_list()`, we could also leave out commas entirely (but cannot mix-and-match separators within a single directive)

```suggestion
    :tags: fort mod map
```

so what we use depends on whether tags could have spaces in them.

Reply:
My hope is that tags can be single words, all lowercase. that way they can be easy to read and easy to remember (not to mention easy to parse)

Reply:
Removed commas